pokemongo_bot/cell_workers/snipe_pokemon.py

```diff
@@ -0,0 +1,119 @@
+import json
+import os
+import re
+import sys
+import time
+
+from . import PokemonCatchWorker
+from pokemongo_bot import logger
+from utils import distance
+
+class SnipePokemon(object):
```

Comment:
This should extend BaseTask which gives you bot and config for free. Take a look at the other workers for an example